docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,53 @@
 Version history
 ===============
 
+5.1.3 (2015 July 21)
+--------------------
+
+* Native C++ updates:
+   - Added cmake superbuild for easier dependency management
+* Bug fixes, including:
+   - Fixed segfault in the `showinf` tool used with the C++ bindings
+   - Allow reading from https URLs
+   - ImageJ
+       - improved performance of displaying ROIs
+   - Command line tools
+       - fixed bfconvert to correctly create datasets with multiple files
+   - Metamorph
+       - improved detection of time series
+       - fixed .nd datasets with variable Z and T counts in each channel
+       - fixed .nd datasets that contain invalid TIFF/STK files
+       - fix dimensions when the number of planes does not match the recorded
+         Z, C, and T sizes
+   - SlideBook
+       - improved native library detection (thanks to Richard Myers)
+   - JPEG
+       - fixed decompression of lossless files with multiple channels (thanks to Aaron Avery)
+   - Imspector OBF
+       - updated to support version 2 files (thanks to Bjoern Thiel)
+   - Imspector MSR
+       - improved detection of Z stacks
+   - PerkinElmer Opera Flex
+       - improved handling of multiple acquisitions of the same plate
+   - Zeiss CZI
+       - fixed error when opening single-file datasets whose names contained
+         `(`  and `)`
```

Comment:
looks a bit confusing rendered, do you want to use '' quote marks you can see in formatted file instead?